lib/gems/pending/appliance_console/cli.rb

```diff
@@ -229,6 +242,20 @@ def set_time_zone
       end
     end
 
+    def set_date_time
+      @new_date = options[:date]
+      @new_time = options[:time]
```

Comment:
These still don't need to be instance variables here and the values still need to be set on the `config_datetime` object before calling `config_datetime.activate` otherwise this will do nothing.